ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -57,6 +53,64 @@ import SwiftUI
     @objc @Published public var layoutType: MSFListCellLayoutType = .automatic
     @objc @Published public var hasDivider: Bool = false
     @objc public var onTapAction: (() -> Void)?
+
+    // Any UIView property will set the value to a type-erased SwiftUI View
+    @objc public var leadingUIView: UIView? {
+        didSet {
+            if let view = self.leadingUIView {
```

Comment:
If the calling code sets the property to nil with the intent of removing the view. How will it work here?
Have you tested that scenario?

Reply:
How about using guard let instead?

I think it falls into the early return scenario in our swift-guide:
https://microsoft.github.io/swift-guide/EarlyReturn.html

```
    @objc public var leadingUIView: UIView? {
        didSet {
            guard let view = leadingUIView else {
                leadingView = nil
                return
            }

            leadingView = AnyView(UIViewAdapter(view))
        }
    }
```